hw/ip/sysrst_ctrl/data/sysrst_ctrl.hjson

```diff
@@ -98,6 +100,52 @@
         }
       ],
     }
+    { name: "ulp_debounce_ctl",
+      desc: "Ultra low power debounce control register",
+      swaccess: "rw",
+      hwaccess: "hro",
+      hwqe: "true",
+      regwen: "REGWEN",
+      resval: "8000",
+      fields: [
+        { bits: "15:0",
+          name: "ulp_debounce_timer",
+          desc: "Configure the debounce timer. 10-200ms, each step is 5us(200KHz clock)",
+        }
+      ],
+    }
+    { name: "ulp_ctl",
+      desc: "Ultra low power control register",
+      swaccess: "rw",
+      hwaccess: "hro",
+      regwen: "REGWEN",
+      resval: "0",
+      fields: [
+        { bits: "0",
+          name: "ulp_enable",
+          desc: "0: disable ULP wakeup feature; 1: enable ULP wakeup feature",
+        }
+        { bits: "1",
+          name: "ulp_fsm_rst",
+          desc: "0: allow the state machine to monitor new wakeup based on ulp_enable setting; 1: clear/reset ULP state machine",
```

Comment:
I think we could simplify the FSM a bit (see further below) such that no explicit `ulp_fsm_rst` bit is needed anymore.
I.e., we could simplify the programming model such that software just has to disable `ulp_enable` in order to reset the state machine and counter.

Reply:
Yes. The programming model is changed. The feature enable bit used to be static (It's enable or disable). Now it has a dual function (feature and FSM start/reset control). Let me check with Andrey.

-Eric

Reply:
Andrey won't be available until Monday(5/17).

-Eric

Reply:
No worries, this is not urgent.

Reply:
@esshiu what is the status of this PR? did you have a chance to sort out the question above? it would be good if we could get this resolved and merged.

Reply:
Right. It's on the top of list. I want to finish it this Friday(6/18).

-Eric